exec/java-exec/src/main/java/org/apache/drill/exec/server/Drillbit.java

```diff
@@ -207,6 +209,28 @@ private void javaPropertiesToSystemOptions() {
   }
 
   /**
+   * Disallow registering drillbit when:
+   * 1. version is unknown;
+   * 2. drillbit with different version has been already registered.
```

Comment:
What if you first DrillBit registered had "bad" version? Somebody would have to bring it down, so others can start?
Should it be (at least) option to have soft enforcement, rather than just hard enforcement.